app/gui2/src/stores/project/index.ts

```diff
@@ -86,7 +86,6 @@ function createLsRpcConnection(clientId: Uuid, url: string, abort: AbortScope):
 function initializeDataConnection(clientId: Uuid, url: string, abort: AbortScope) {
   const client = createDataWebsocket(url, 'arraybuffer')
   const connection = new DataServer(clientId, client, abort)
-  abort.handleDispose(connection)
```

Comment:
Why is this line removed?

Reply:
i think DataServer already calls the exact same method on the same AbortScope, right?

Reply:
Right.